### ISSUE TITLE
Setting node_version to 14.16.0 (LTS) does not work

### ISSUE THREAD

**Issue by flolu** [bazelbuild/rules_nodejs#2581]
# 🐞 bug report

### Description

I wan to specify a custom [`node_version`](https://github.com/bazelbuild/rules_nodejs/blob/e92bc7ae6476d8d4c079a9947b59c197a2d84ff4/docs/Built-ins.md#node_version) for my project. The default is `12.13.0`, but I rather want to use the long term supported version, which currenlty is `14.16.0`

So I specified this in my `WORKSPACE`
```
node_repositories(node_version = "14.16.0")
```

But it does not work (see error below).

Strangely, any other version works fine. For example:
```
node_repositories(node_version = "14.15.5")
```

## 🔬 Minimal Reproduction

https://github.com/flolu/node14-unknown-version

## 🔥 Exception or Error

<pre><code>
ERROR: no such package '@nodejs_linux_amd64//': Unknown NodeJS version-host 14.16.0-linux_amd64
</code></pre>

## 🌍  Your Environment

**Operating System:**

Ubuntu 20.04 LTS

**Comment by mattem** [comment]
Fixed via #2582 
We have a script to update the list of versions available, and a GH action that should fire PRs to keep it up to date which was failing to run, fixed that up too so this is no longer a manual process, #2583

**Comment by mattem** [comment]
closed via #2583
